.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'canary*'
+
+name: Create Canary Release
+
+jobs:
+  build:
+    name: Create Canary Release
+    runs-on: ubuntu-20.04
+    container:
+        image: prosty/sf2000-multicore-builder:v1.1-release
```

Comment:
**note:**
I've pushed the image out to my dockerhub, as it's the same image as my GB300 builder. But you can as well upload it to yours @madcock . Simply:

```
docker build -t {DH_USERNAME}/sf2000-multicore-builder:v1.1 .
docker tag {DH_USERNAME}/sf2000-multicore-builder:v1.1 {DH_USERNAME}/sf2000-multicore-builder:v1.1-release
docker push {DH_USERNAME}/gb300-multicore-builder:v1.1-release
```